.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -126,6 +126,7 @@ fun applyDefaults(model: CIBuildModel, buildType: BaseGradleBuildType, gradleTas
     var gradleParameterString = gradleParameters.joinToString(separator = " ")
```

Comment:
Maybe `gradleParameters` could be a function which takes the `os` and whether to use the daemon or not as parameters. Then we won't need to do any String replacements here any more. WDYT?

Reply:
Sounds like a good refactoring. I'll do it.